cerberus-org/cerberus-app
# Sign up tests

This PR encompasses 'c' of Create sign up component.

**Test**
Steps:

1. Fill in form
2. Select 'Submit' or press enter
3. Check the console
4. Ensure http response was returned and logged

Note: I would like to make a separate PR to handle http errors properly
